pkgs/top-level/all-packages.nix

```diff
@@ -30116,6 +30116,10 @@ with pkgs;
     inherit (pkgs.darwin.apple_sdk.frameworks) Accelerate AudioUnit CoreAudio CoreMIDI;
   };
 
+  csound-plugins = callPackage ../applications/audio/csound/plugins.nix {
```

Comment:
Given that by-name is mandatory for new packages, please migrate it.

Hint: use passthru.

Reply:
OK, will do.

Reply:
Done.